src/services/forecastPeriod.service.js

```diff
@@ -23,4 +24,18 @@ function setForecastPeriod(number){
     }catch(e){
         console.log(e);
     }
+}
+
+async function getForecastByLatLng(latitude,longitude){
+    const key = '6866075b0c665a854c8731d19df394eb';
+    const url = `https://api.openweathermap.org/data/2.5/forecast?lat=${latitude}&lon=${longitude}&appid=${key}`;
+    try{
+        const result = await fetch(url,{
+            method: 'get'
+        });
+        const json = await result.json();
+        return json;
+    }catch(e){
+        return {error: e};
```

Comment:
As far as you don't handle errors in action - don't catch errors here. It degrades readability of the code.